openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/edgefilters/core/SpeedCoreEdgeFilter.java

```diff
@@ -0,0 +1,39 @@
+/*  This file is part of Openrouteservice.
+ *
+ *  Openrouteservice is free software; you can redistribute it and/or modify it under the terms of the
+ *  GNU Lesser General Public License as published by the Free Software Foundation; either version 2.1
+ *  of the License, or (at your option) any later version.
+
+ *  This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Lesser General Public License for more details.
+
+ *  You should have received a copy of the GNU Lesser General Public License along with this library;
+ *  if not, see <https://www.gnu.org/licenses/>.
+ */
+package org.heigit.ors.routing.graphhopper.extensions.edgefilters.core;
+
+import com.graphhopper.routing.EdgeIteratorStateHelper;
+import com.graphhopper.routing.util.EdgeFilter;
+import com.graphhopper.routing.util.FlagEncoder;
+import com.graphhopper.storage.GraphStorage;
+import com.graphhopper.util.EdgeIteratorState;
+import org.heigit.ors.routing.graphhopper.extensions.storages.GraphStorageUtils;
+import org.heigit.ors.routing.graphhopper.extensions.storages.SpeedStorage;
+
+
+public class SpeedCoreEdgeFilter implements EdgeFilter {
```

Comment:
There doesn't seem to be any instance of `SpeedCoreEdgeFilter` created across ORS codebase. How is the core-based routing with dynamic speeds supposed to work without including the corresponding edges in core? 🤔

Reply:
The project for this algorithm includes support for dynamic speed edges, but not the creation of these edges. That is development work on the side of the customer. 
We do not know yet how the edges that go into the core are defined, so it is impossible to create this, anyway.

Reply:
Thanks for the clarification! Do you think we could have a more descriptive name for this filter, such as for example `VariableSpeed...` or `ExternalSpeed...`?

Reply:
Done